docs/sources/search/_index.md

```diff
@@ -14,111 +16,24 @@ title: Search
-{{% /admonition %}}
-
-## Search dashboards using panel title
-
-You can search for a dashboard by the title of a panel that appears in a dashboard.
-If a panel's title matches your search query, the dashboard appears in the search results.
-
-This feature is available by default in Grafana Cloud and in Grafana OSS v9.1 and higher, you access this feature by enabling the `panelTitleSearch` feature toggle.
-For more information about enabling panel title search, refer to [Enable the panelTitleSearch feature toggle.](#enable-panelTitleSearch-feature-toggle)
-
-The following image shows the search results when you search using panel title.
-
-{{< figure src="/static/img/docs/v91/dashboard-features/search-by-panel-title.png" width="700px" >}}
-
-### Enable the panelTitleSearch feature toggle
-
-Complete the following steps to enable the `panelTitleSearch` feature toggle.
-
-**Before you begin:**
-
-- If you are running Grafana Enterprise with RBAC, enable [service accounts]({{< relref "../administration/service-accounts/" >}}).
-
-**To enable the panelTitleSearch feature toggle:**
-
-1. Open the Grafana [configuration file]({{< relref "../setup-grafana/configure-grafana/#configuration-file-location" >}}).
-
-1. Locate the [feature_toggles]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}) section.
-
-1. Add the following parameter to the `feature_toggles` section:
-
-   ```
-   [feature_toggles]
-   # enable features, separated by spaces
-   enable = panelTitleSearch
-   ```
-
-1. Save your changes and restart the Grafana server.
-
-## Filter dashboard search results by tag(s)
-
-Tags are a great way to organize your dashboards, especially as the number of dashboards grow. You can add and manage tags in dashboard `Settings`.
-
-When you select multiple tags, Grafana shows dashboards that include all selected tags.
-
-To filter dashboard search result by a tag, complete one of the following steps:
-
-- To filter dashboard search results by tag, click a tag that appears in the right column of the search results.
-
-  You can continue filtering by clicking additional tags.
-
-- To see a list of all available tags, click the **Filter by tags** dropdown menu and select a tag.
-
-  All tags will be shown, and when you select a tag, the dashboard search will be instantly filtered.
-
-{{% admonition type="note" %}}
-When using only a keyboard, press the `tab` key and navigate to the **Filter by tag** drop-down menu, press the down arrow key `▼` to activate the menu and locate a tag, and press `Enter` to select the tag.
-{{% /admonition %}}
+Use the command palette to search all of Grafana and to initiate key actions from any page in the application.
 
 ## Command palette
 
-The command palette enables you to:
+The command palette allows you to:
 
 - Search for and open dashboards and folders
 - Create dashboards and alert rules
 - Locate pages within Grafana
 - Change the theme to dark or light
+- Run Sift investigations
```

Comment:
I did not know about this one. I checked in ops, and I found it using the command palette, but [in the documentation](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/sift/sift/#sift-investigations), we don't have the command palette as a way to run them. Is that on purpose?

Reply:
> I did not know about this one. I checked in ops, and I found it using the command palette, but [in the documentation](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/sift/sift/#sift-investigations), we don't have the command palette as a way to run them. Is that on purpose?

Good question! @alyssawada, do you have any insight here?

Reply:
> > I did not know about this one. I checked in ops, and I found it using the command palette, but [in the documentation](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/sift/sift/#sift-investigations), we don't have the command palette as a way to run them. Is that on purpose?
> 
> Good question! @alyssawada, do you have any insight here?

I confirmed with @JohnnyK-Grafana (who works on the machine learning content) that you can indeed launch a sift investigation from the command palette. He's going to update those docs. So we're all good!